modules/http4s-munit-testcontainers/src/main/scala/munit/HttpFromContainerSuite.scala

```diff
@@ -68,7 +68,8 @@ abstract class HttpFromContainerSuite
             json
               .replaceAll("""(\"\w+\") : """, Console.CYAN + "$1" + Console.RESET + " : ")
               .replaceAll(""" : (\".*\")""", " : " + Console.YELLOW + "$1" + Console.RESET)
-              .replaceAll(""" : (\d+)""", " : " + Console.GREEN + "$1" + Console.RESET)
+              .replaceAll(""" : (-?\d+.\d+)""", " : " + Console.GREEN + "$1" + Console.RESET)
```

Comment:
If you want to target literal dot `.` you should use `\.` instead, since `.` in markdown means any character.

```suggestion
              .replaceAll(""" : (-?\d+\.\d+)""", " : " + Console.GREEN + "$1" + Console.RESET)
```